package.json

```diff
@@ -52,13 +52,13 @@
     "helmet": "^6.0.1",
     "http-graceful-shutdown": "^3.1.13",
     "ioredis": "^5.3.1",
-    "is-ip": "^5.0.0",
     "isbot": "^3.6.6",
     "nodemailer": "^6.9.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "samlify": "^2.8.8",
     "tiny-invariant": "^1.3.1",
+    "validator": "^13.9.0",
```

Comment:
I suspect you should add `@types/validator` too (devDependency) since this is a definitely typed package

Reply:
Will be added